Jenkinsfile

```diff
@@ -29,35 +44,32 @@ pipeline {
                 sh 'nohup service tango-db start'
                 sh 'pip install . -U --user'
                 sh 'pip install nose_xunitmp --user'
```

Comment:
Are we running tests on multiple CPU's? (As per [README](https://github.com/Ignas/nose_xunitmp/blob/master/README.rst)) if not why do we need to install `nose_xunitmp` when we could just use nosetests `--with-xunit` arg.

Reply:
It was required to actually produce a `nosetests.xml` report that isn't empty. 

We're probably not running multi-core tests but TANGO stuff is definitely multi-threaded.

Reply:
Ok cool!